examples/flutter_gallery/android/AndroidManifest.xml

```diff
@@ -8,6 +8,19 @@
     <uses-sdk android:minSdkVersion="14" android:targetSdkVersion="21" />
     <uses-permission android:name="android.permission.INTERNET"/>
 
+    <compatible-screens>
```

Comment:
Maybe add a comment for what we're doing here? Perhaps "White-listing phones, until such time as Flutter's widgets and layouts have been tested for tablets."